mlflow/protos/unity_catalog_oss_service.proto

```diff
@@ -26,4 +26,95 @@ service UnityCatalogService {
       visibility: PUBLIC,
     };
   }
+
+  rpc updateRegisteredModel(UpdateRegisteredModel) returns (UpdateRegisteredModel.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "PATCH",
+        path: "/unity-catalog/models/{full_name}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc deleteRegisteredModel(DeleteRegisteredModel) returns (DeleteRegisteredModel.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "DELETE",
+        path: "/unity-catalog/models/{full_name}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc getRegisteredModel(GetRegisteredModel) returns (GetRegisteredModel.Response) {
+    option idempotency_level = IDEMPOTENT;
```

Comment:
What is this flag?

Reply:
The flag is used here: https://github.com/databricks/universe/blob/master/managed-catalog/api/endpoints/service.proto#L1163. 

I think its just some way of indicating that calling the operation doesn't cause any changes (i.e. you can call the same thing 100 times with no effect on the underlying object state https://stackoverflow.com/questions/1077412/what-is-an-idempotent-operation). 

Both getRegisteredModel and getModelVersion are classified correctly as Idempotent, but it might be unnecessary to label them. Let me know what you think, I can remove it if you think we should.